thirdparty/mupdf/CMakeLists.txt

```diff
@@ -10,7 +10,12 @@ enable_language(C)
 assert_var_defined(HOSTCFLAGS)
 assert_var_defined(HOSTCC)
 assert_var_defined(OS)
-set(BUILD_CMD_GENERATE sh -c "env CFLAGS=\"${HOSTCFLAGS}\" $(MAKE) -j${PARALLEL_JOBS} generate build=\"release\" CC=\"${HOSTCC}\" verbose=1")
+if($ENV{KODEBUG})
+    set(MUPDF_BUILD_TYPE "debug")
+else()
+    set(MUPDF_BUILD_TYPE "release")
+endif()
+set(BUILD_CMD_GENERATE sh -c "env CFLAGS=\"${HOSTCFLAGS}\" $(MAKE) -j${PARALLEL_JOBS} generate build=\"${MUPDF_BUILD_TYPE}\" CC=\"${HOSTCC}\" verbose=\"yes\"")
```

Comment:
Is `1` and `yes` equivalent here or was the `1` not doing anything?

Reply:
The Makefile is explicitly doing `ifneq "$(verbose)" "yes"``, so, yeah, if the intent was to get verbose builds, that wasn't doing the trick ;).

Whether we actually *want* a verbose build here is debatable, though ;).
You know my position on the subject, but I'm perfectly okay with switching that back to "no" here ;).

Reply:
Except I remember this check being somewhat counter-intuitive...

I think we *want* this to no, because it's actually *more* silent when set to yes?

Reply:
Verbose all the way as far as I'm concerned. It's just that too much of it can cause trouble with one of the CI services (definitely Travis and GitLab, possibly CircleCI as well).

Reply:
I think at least one of them (not Travis, certainly not GitLab) might do something more akin to a terminal emulator (i.e., just a limited amount of scrollback), which sounds potentially a lot more useful to catch the error.